components/script/timers.rs

```diff
@@ -502,7 +502,7 @@ impl JsTimerTask {
             },
             InternalTimerCallback::FunctionTimerCallback(ref function, ref arguments) => {
                 let arguments = self.collect_heap_args(arguments);
-                let _ = function.Call_(this, arguments, Report);
+                let _ = function.Call_(this, arguments, Report);  // TODO this is of type T, I can't locate where function is defined (somewhere in bind gen?)
```

Comment:
Function is generated by CodegenRust.py due to components/script/dom/webidl/Function.webidl

Reply:
I believe I managed to hack it in some way and let Servo compile. I wanted to make the function to accept `#[must_root]` type, I guess.